package.json

```diff
@@ -144,6 +145,11 @@
     },
     "coverageDirectory": "./coverage/"
   },
```

Comment:
@tmorehouse I'am not sure if this is a good idea for the time it costs to push the changes. At least we are only using `prettier` for the docs and not the code itself.